galaxy_ng/app/migrations/0028_move_perms_to_roles.py

```diff
@@ -337,7 +343,11 @@ def migrate_group_permissions_to_roles(apps, schema_editor):
     # Group Permissions
     for group in Group.objects.filter(name__ne="system:partner-engineers"):
         group_roles.extend(get_global_group_permissions(group, Role, GroupRole, Permission))
-        group_roles.extend(get_object_group_permissions(group, Role, GroupRole, ContentType, Permission))
+
+        # Skip migrating object permissions if guardian is not installed.
+        if is_guardian_table_avaiable:
+            group_roles.extend(
+                get_object_group_permissions(group, Role, GroupRole, ContentType, Permission))
```

Comment:
When would this table not exist? is that to cover in case the migration is run again, or something else?

Reply:
that table doesn't exist if you're installing a new instance of hub after guardian was removed (ie not upgrading). I set it up so it only skips the object roles and not the model roles because this lets us run the unit tests for the model roles migrations, which is useful

Reply:
Would this cover fresh installs after guardian has been removed?

Reply:
The skipping after guardian removal makes sense, but why is it useful to keep running unit tests for the model roles migrations?

Reply:
The unit for the migration helped me catch a few bugs and I kinda don't want to get rid of them just yet in case we need to make other changes before the feature branch is merged. I we could remove them in a followup pr. WDYT?